src/status_im/ui/screens/db.cljs

```diff
@@ -42,7 +45,7 @@
              :tags                       []
              :sync-state                 :done
              :wallet.transactions        constants/default-wallet-transactions
-             :wallet                     {:send-transaction transaction-send-default}
+             :wallet                     {:send-transaction (transaction-send-default)}
```

Comment:
why it became a function?

Reply:
I did this change to prepare for the move to a computed gas price (#3241)